app/templates/search/_briefs_results.html

```diff
@@ -18,7 +18,7 @@ <h2 class="search-result-title">
             {{ brief.lotName }}
         </li>
         <li class="search-result-metadata-item">
-            {{ brief.frameworkName }}
+            {{ brief.specialistRole }}
```

Comment:
Are we happy not showing anything here at all if it's not on the specialists lot?  Because for all other lots this won't exist and we end up with an empty list item.